src/Kernel.package/AsciiCharset.class/class/toLowercase..st

```diff
@@ -1,9 +1,9 @@
 casing
 toLowercase: aCharacter
-	"(AsciiCharset new toLowercase: $A) >>> $a.
-	(AsciiCharset new toLowercase: $a) >>> $a.
-	(AsciiCharset new toLowercase: $!) >>> $!"
+	"(AsciiCharset toLowercase: $A) >>> $a.
+	 (AsciiCharset  toLowercase: $a) >>> $a.
+	 (AsciiCharset  toLowercase: $!) >>> $!"
+
 	(aCharacter between: $A and: $Z)
 		ifFalse: [ ^ aCharacter ].
-	
-	^ Character value: (aCharacter asciiValue + $a asInteger - $A asInteger )
\ No newline at end of file
+	^ Character value: aCharacter asciiValue + $a asInteger - $A asInteger
```

Comment:
Given that the receiver is AsciiCharset, it might be more meaningful to bitOr: 16r20. (Not that many people realize how teletypes worked anymore.)

Reply:
I think that either implementation is fine.  I didn't change this code at all — I just fixed the executable comment so that it actually executes.

Personally, I visualise "toLowercase" as moving a block of Ascii codes down in the character chart.  think that subtraction is a clear way of conveying this, so I'm uninclined to change it.

Reply:
Yes, I prefer the clarity of the current solution. We could maybe change to the other one if it has a **REAL** impact in performance and we keep the old solution as a comment.